src/main/java/com/WWI16AMA/backend_api/Application.java

```diff
@@ -29,6 +31,10 @@
 @SpringBootApplication
 public class Application extends SpringBootServletInitializer {
 
+    //Speichert eine MemberId aus generateSomeMembers, um diese für weitere erstellungen nutzen zu können
+    // (bsp. generateSOmePlaneLogs())
+    private static int memberId = 0;
```

Comment:
würd ich nicht so machen, sondern `generateSomePlaneLogs(Plane plane, Member member)` machen, also den Member als Parameter mitgeben. Is glaub ich n bissel _sauberer_

Reply:
Geändert, er nimmt jetzt das plane- und member-Repo als Parameter. Nimmt ein zufälliges Member und das Plane mit der id 1.